src/gui/folder.cpp

```diff
@@ -454,19 +453,21 @@ void Folder::setSyncPaused(bool paused)
     _definition.paused = paused;
     saveToSettings();
 
+    emit syncPausedChanged(this, paused);
     if (!paused) {
         setSyncState(SyncResult::NotYetStarted);
     } else {
         setSyncState(SyncResult::Paused);
     }
-    emit syncPausedChanged(this, paused);
-    emit syncStateChange();
     emit canSyncChanged();
 }
 
 void Folder::setSyncState(SyncResult::Status state)
```

Comment:
I guess this could be renamed to `changeSyncState` to suggest that this emits a signal, too.

Reply:
Thats not how Qt api is named